docs/DeveloperGuide.md

```diff
@@ -395,8 +409,8 @@ MSS:
    Extensions:
 
 - 1a. Item does not exist
-  1a1. Foodrem displays an error.
-  1a2. Foodrem asks the user if they want to try again
+  1a1. FoodRem displays an error.
+  1a2. FoodRem asks the user if they want to try again
```

Comment:
Missing line breaks. Probably better to standardize and have each step on a new line instead. 

Current it looks something like this:
```
- 1a. Item does not exist 1a1. FoodRem displays an error. 1a2. FoodRem asks the user if they want to try again
```